spec/models/link_spec.rb

```diff
@@ -203,7 +203,7 @@
 
     it "sets the link status, link warnings and last checked time to an existing homepage url status, warnings and link last checked time" do
       @local_authority = create(:local_authority, status: "broken", link_warnings: @warnings, link_last_checked: "2016-07-14 11:34:09 +0100")
-      @link = create(:link, url: "http://example.com/thing", status: "ok", link_last_checked: Time.now)
+      @link = create(:link, url: "http://example.com/thing", status: "ok", link_last_checked: Time.zone.now, local_authority: @local_authority)
```

Comment:
This took me a little while to appreciate!

Reply:
Yeah I tried explaining in the commit message, but not sure how clear it is (lots of repetition of Local Authority 🙈)

Reply:
That's fine - esoteric factory behaviour is hardly something to fix in this PR :-).